### PR TITLE
Save power by adding delay between network requests in background.

### PR DESCRIPTION
**Pull request by mbarashkov**
This is a remake of https://github.com/Wunderfitz/harbour-fernschreiber/pull/572

### CODE REVIEW

File: qml/pages/OverviewPage.qml [hunk 1]
```diff
@@ -88,7 +88,8 @@ Page {
     }
     Timer {
         id: updateSecondaryContentTimer
-        interval: 600
+        interval: 1500
+        repeat: false
```
Comment (monich) [inline qml/pages/OverviewPage.qml:92]: There's no need for this, `repeat` defaults to `false`

Comment (mbarashkov) [reply qml/pages/OverviewPage.qml:92]: > There's no need for this, `repeat` defaults to `false`

Above this another method states "repeat: false" expicitly. I assumed that's a code style. After all it's better readable this way..

Comment (monich) [reply qml/pages/OverviewPage.qml:92]: And is this necessary at all, to increase the interval? This is a one-time action, it won't have any impact on power usage or anything like that.

Comment (mbarashkov) [reply qml/pages/OverviewPage.qml:92]: Well, yes, it's logically unrelated, I've just noticed it improves app startup performance.. Ok, I will redo the PR again.

**Comment by mbarashkov** [comment]
@monich change removed in https://github.com/Wunderfitz/harbour-fernschreiber/pull/574

**Pull request closed without merging**